docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -1163,4 +1176,4 @@ For each generated "Trend" field value calculation function can be selected. Def
 
 {{% /docs/reference %}}
 
-[Data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
+[data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
```

Comment:
This removal of the capitalization **might** be an issue, depending on what you're trying to do. When the Hugo processor turns this markdown into html, this syntax will add a hyperlink to all `Data frames` strings in the output, but since it's case sensitive, `data frames` will not be given any link. This change will assure that all `data frames` strings will get the link, but `Data frames` won't. But it looks like in the text, `data frames` is used as a common noun, but `Data frames` is used as a proper noun - which is what should be linked.

Reply:
This change is odd since this file is auto generated. I can manually change it, but it will probably change back when we auto generate the docs again.

Reply:
Looks like this resolved in your most current code. (b^_^)b